spec.md

```diff
@@ -426,6 +428,7 @@ how Platforms might expose these values to their users.
 | free | boolean | When false, Service Instances of this plan have a cost. The default is true. |
 | bindable | boolean | Specifies whether Service Instances of the Service Plan can be bound to applications. This field is OPTIONAL. If specified, this takes precedence over the `bindable` attribute of the service. If not specified, the default is derived from the service. |
 | schemas | [Schemas](#schemas-object) | Schema definitions for Service Instances and bindings for the plan. |
+| deprecation | [Deprecation](#deprecation-object) | Contains details of the deprecation status of the plan. This field is OPTIONAL. If specified, the plan is deprecated. See [deprecating services and plans](#deprecating-services-and-plans) for more information. |
```

Comment:
ditto: remove "This field is OPTIONAL."

Reply:
oh man we do say it on 'bindable".  Geez, why? we should remove that.

Reply:
gone.